### ISSUE TITLE
Config file for setting table names

### ISSUE THREAD

**Issue by michaeljhopkins** [spatie/laravel-permission#5]
I think it's likely pretty common that a lot of us already have tables (from some other 3rd party acl) to do most of what this migrates. In interest of keeping db changes to a minimum, it'd be ideal for something like this

```php
'tables' => [
    'permissions' => env('PERMISSIONS_TABLE','permissions'),
    'user_has_permissions' => env('USER_HAS_PERMISSIONS_TABLE','user_has_permissions'),
]
```
and so on with the rest of the tables. 

Short issue, I have every table you're creating already, just named differently

**Comment by freekmurze** [comment]
I'll add a config file to the package to allow naming of the tables in the near future.

**Comment by michaeljhopkins** [comment]
Working on a PR just to get the functionality there. However you'll likely need to touch it up a bit

**Comment by michaeljhopkins** [comment]
see if that change works for you!

cheers
-m

**Comment by freekmurze** [comment]
I've added some comments to your PR.

**Comment by freekmurze** [comment]
Solved in v1.2.0 of the package
